docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.deploy.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
What is this file? i've never heard of a deploy.php file nor do I understand how the function is intended to be used. I don't find any documentation for deploy hooks.

If this is just a simple update hook, move it to an install or post_update file

Reply:
@pookmish  Deploy hooks are part of drush and are executed after the configuration is imported. There's more information here:
- https://www.drush.org/12.x/deploycommand/
- https://www.hashbangcode.com/article/drupal-9-different-update-hooks-and-when-use-them

We consider is the best way for cases like this, when the db updates depend on configuration that's exported in the same PR. However, if you consider that's better to use an update hook, please let us know and we can change it.

Reply:
It looks like deploy hooks are only part of drush and not part of core. There [is an issue](https://www.drupal.org/project/drupal/issues/3206497) in the queue to add them to core, but there's not a lot of traction on it. It seems really useful but I'm a little concerned it's only drush supported.

Reply:
I agree with Joe. To add to that, we would have to modify the deployment process to add the drush command. Let's move it to an update/post_update hook.

Reply:
@joegl @pookmish Done, PR ready for review again. Thanks!